index.html

```diff
@@ -15,5 +15,155 @@
+        <!-- End of Hamburger menu  -->
+
+        <!-- Navbar -->
+          <div>
+            <ul class='navigation-list'>
+              <li class='navigation-item'>
+                <a class='active navigation-link' href="#">Meet Karma</a>
+              </li>
+              <li class='navigation-item'>
+                <a class='navigation-link' href="#">How it Works</a>
+              </li>
+              <li class='navigation-item'>
+                <a class='navigation-link' href="#">Store</a>
+              </li>
+              <li class='navigation-item'>
+                <a class='navigation-link' href="#">Blog</a>
+              </li>
+              <li class='navigation-item'>
+                <a class='navigation-link' href="#">Help</a>
+              </li>
+              <li class='navigation-item'>
+                <a class='navigation-link' href="#">Login</a>
+              </li>
+            </ul>
+          </div>
+        <!-- End of Navbar div  -->
+    </header>
+
+<!-- Main content -->
+    <div class='content'>
+        <main class='main'>
+                <h1>Introducing Karma</h1>
+                <p class='h1Paragraph'>Bring Wifi with you, Everywhere you go</p>
+           
+                <button>Learn More</button>
+        </main> 
+<!-- H2 for articles -->
+        <h2 class ='article-h2'>Everyone needs a little Karma.</h2>
+<!-- articles -->
+        <div class='articles'>
+          <!-- Devices -->
+            <article>
+                <img class='article-img' src="img/icon-devices.svg" alt="desktop computer image"/>
+                <a href="#"> Internet for all devices</a>
+            </article>
+            
+          <!-- productivity   -->
+            <article> 
+                <img class='article-img' src="img/icon-coffee.svg" alt="hot coffee-cup"/>
+                <a href="#"> Boost your productivity</a>
+            </article>
+          <!-- Refil -->
+            <article>
+                <img class='article-img' src="img/icon-refill.svg" alt="refil image of a mobile device/">
+                <a href="#"> Pay as You Go</a>
+            </article>
+
+        </div>  <!-- End of article div -->
+
+    <!-- extended-article  -->
+        <div class='extended-article'>
+            <div class='image'>
+                <img class='image-size' src="additional/feature-img.png" 
+                alt="an woman sitting down back smilling looking at her ipad screen next to a wooden table">
+            </div>
+            <div class='extended-article-textsAndButton'>
+                  <em class='h2-of-extended-article'>
+                    <h2>Whereever I am, </h2>
+                    <h2>I just don't worry about my connection</h2>
+                    <h2>anymore!</h2>
+                  </em> 
+                <button>Get Karma today</button>
+            </div>
+        </div>
+    <!-- End of extended-article  -->
+
+    </div><!-- End of content div -->
+
+<!-- hr and line break used -->
```

Comment:
I'd suggest comments like these shouldn't be necessary. As much as possible try and make your code as clear as possible so that you don't need to use comments as much.